app/app.go

```diff
@@ -114,6 +121,8 @@ func NewInitApp(
 		oracle.StoreKey,
 		ethbridge.StoreKey,
 		clp.StoreKey,
+		distr.StoreKey,
+		slashing.StoreKey,
```

Comment:
Should we implement slashing for peggy?

Reply:
What do you mean? This is for the sifchain? and there will effort both at this point.